lib/board.js

```diff
@@ -53,6 +53,10 @@ Board.prototype.createIcons = function () {
   });
 };
 
+Board.prototype.currentTurn = function (color) {
+  return this.game.turn === color;
+};
```

Comment:
It seems a little squirrelly to me that that `currentTurn`—a name that sounds like a property—is actually a method and takes a color as argument. It's worth considering renaming this method to something that explains the action being done in the code.